src/ontology/reports/icd10who.synonyms.confirmed.robot.tsv

```diff
@@ -1,4 +1,4 @@
-mondo_id	mondo_label	synonym_scope	synonym_scope_mondo	synonym	synonym_exact	synonym_broad	synonym_narrow	synonym_related	source_id	source_label	synonym_type	mondo_evidence	case
-ID					A oboInOwl:hasExactSynonym	A oboInOwl:hasBroadSynonym	A oboInOwl:hasNarrowSynonym	A oboInOwl:hasRelatedSynonym	>A oboInOwl:hasDbXref		>AI oboInOwl:hasSynonymType		
-MONDO:0005027	epilepsy	oio:hasExactSynonym		epilepsy	epilepsy				ICD10WHO:G40	Epilepsy	http://purl.obolibrary.org/obo/mondo#CLINGEN_LABEL		
-MONDO:0007950	mastocytosis	oio:hasExactSynonym		mastocytosis	mastocytosis				ICD10WHO:Q82.2	Mastocytosis			
+mondo_id	mondo_label	synonym_scope	synonym_scope_mondo	synonym	synonym_exact	synonym_broad	synonym_narrow	synonym_related	source_id	source_label	synonym_type	synonym_type_mondo	mondo_evidence	case
+ID					A oboInOwl:hasExactSynonym	A oboInOwl:hasBroadSynonym	A oboInOwl:hasNarrowSynonym	A oboInOwl:hasRelatedSynonym	>A oboInOwl:hasDbXref		>AI oboInOwl:hasSynonymType SPLIT=|			
+MONDO:0005027	epilepsy	oio:hasExactSynonym		epilepsy	epilepsy				ICD10WHO:G40	Epilepsy	http://purl.obolibrary.org/obo/mondo#CLINGEN_LABEL	http://purl.obolibrary.org/obo/mondo#CLINGEN_LABEL		
```

Comment:
CLINGEN_LABEL is only provenance. It should absolutely not appear in an active ROBOT template column!

<img width="634" alt="image" src="https://github.com/user-attachments/assets/09551f51-0268-4946-a8a6-bb3111773331">

Reply:
Agreed. This also occurs on Line 13434 where the source is DOID and perhaps other lines. 

The synonym type `CLINGEN_LABEL` is only a synonym type in Mondo so it's not clear to me why it is showing up in the column `synonym_type`, which I understand should only have source synonym type information.

Reply:
This has to do with the merging of synonym_type and synonym_type_mondo. I have another comment about that.

But it looks like that concern is not founded, so I will not merge these and then this will go away.